cf-oscore/src/main/java/org/eclipse/californium/oscore/OSCoreResource.java

```diff
@@ -34,6 +35,7 @@ public class OSCoreResource extends CoapResource {
 
 	public OSCoreResource(String name, final boolean isProtected) {
 		super(name);
+		getAttributes().setOscoreOnly();
```

Comment:
Is there a relation betwenn "isProtected" and "Oscore Only"?
I searched for references to is Protected but could not find something.

Reply:
Yes, great point, I missed that. At a minimum _setOscoreOnly()_ should only be invoked when _isProtected_ is set to true. Since _isProtected_ is the flag that is actually controlling if the resource is only accessible with OSCORE, it's not enough that the resource type is OSCoreResource.

I will take care of this.

Reply:
Took care of only setting the OSCORE link target attribute if _isProtected_ is true in commit b9996e21d07b0c0bbe1ce45962ad54bf5cb03aff